pipeline/run-pipeline.R

```diff
@@ -0,0 +1,17 @@
+# Fetch data
+source("scripts/fetch-data/fetch_data.R")
+
+# Process data
+source("scripts/process-data/fitbitactivitylogs.R")
+source("scripts/process-data/fitbitdailydata.R")
+source("scripts/process-data/fitbitintradaycombined.R")
+source("scripts/process-data/fitbitsleeplogs.R")
+source("scripts/process-data/healthkitv2samples.R")
```

Comment:
healthkitv2activitysummaries?

Reply:
We decided to not include any variables from `dataset_healthkitv2activitysummaries` for the 2024 Q1 release. However, the code is ready to be included for that dataset if we do decide to include any of its variables in the future.

[See this comment on the related JIRA issue](https://sagebionetworks.jira.com/browse/RMHDR-242?focusedCommentId=199542)